aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -46,6 +46,14 @@ else
 	curl --silent -H "Content-Type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d '{"name":"core-user-rule","description": "Core User is Admin of all containers","role":"administrator","resources":{"containers":["*"],"images":["*"],"volumes":["*"],"networks":["*"]},"accessors":{"users":["core"]}}' $WEB_URL/adminrules
 fi
 
+EXISTING_LABEL=$(curl --write-out %{http_code} --silent --output /dev/null -H "$HEADER: Bearer $TOKEN" $WEB_URL/settings/labels/production%20approved)
+
+if [[ "$EXISTING_LABEL" == "200" ]]; then
+	log "production approved label exists..."
+else
+	curl --silent -H "Accept: application/json" -H "Content-type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d '{"name": "production approved", "description": "production approved images"}' $WEB_URL/settings/labels
```

Comment:
You mean these steps ? - Create a label (this is done)/ Attach it to image(?): https://docs.aquasec.com/v2.0/docs/labels
Creating a host batch install(?):
https://docs.aquasec.com/v2.0/docs/batch-install (there is a policy on the host for allowed labels).